shared/@hedvig-ui/HotkeyHint/index.tsx

```diff
@@ -78,6 +79,32 @@ export interface HotkeyHintProps extends HTMLMotionProps<'div'> {
   remind?: boolean
 }
 
+export const toastReminder = ({ keys }: { keys: Key[] }) =>
+  toast(
+    () => (
+      <div>
+        <span style={{ marginRight: '0.35rem' }}>Please, try to use </span>
```

Comment:
I think we can skip the word "Please".
```suggestion
        <span style={{ marginRight: '0.35rem' }}> Try to use </span>
```

Reply:
Maybe, instead of having the setting. We should store some cookie as well that gives a "cooldown" for these till the next day.